content/pages/constitution.md

```diff
@@ -168,9 +170,33 @@ computing.
     2. To design all media circulated by the committee including: posters,
        social media profiles and the official website.
 
-All committee memebers are free to delegate their responsibilities to others as
-as they see fit, but are ultimately responsible for the completion of their
-assigned duties.
+9.  All committee memebers are free to delegate their responsibilities to others
+    as they see fit, but are ultimately responsible for the completion of their
+    assigned duties.
+
+10. All committee members are responsible to support their fellow members and
```

Comment:
I am +1 to the idea of this section but it seems a little too detailed for me. Committee members falling behind definitely need to be supported, but also, I think the specific procedure to how best support that member should be left to the discretion of the people involved in the situation. The constitution should only give an outline with options, not step by step instructions.

ie. _"All members are responsible to support their fellow members...supporting members can reach out privately and offer assistance or bring up the responsibility to other members of the committee, as the situation calls for"_